Dizzy/Web Service/FirebaseWebService.swift

```diff
@@ -33,12 +33,18 @@ final class FirebaseWebService: WebServiceType {
                                                 completion: @escaping (Result<Response>) -> Void) {
         databaseReference.child(resource.path).observe(DataEventType.value) { snapshot in
             do {
-                guard let json = snapshot.value as? [String: Any] else {
+            
+                guard let jsonsArray = snapshot.value as? [String: Any] else {
                     completion(Result<Response>.failure(WebServiceError.noData))
                     return
                 }
                 
-                let data = try JSONSerialization.data(withJSONObject: json, options: .prettyPrinted)
+                var jsonToParse = [[String: Any]]()
+                for (_,value) in jsonsArray {
+                    jsonToParse.append(value as! [String : Any])
+                }
```

Comment:
i think it will be better if you extract the code from line 35 to line in to separate function called `getJsonToParse(From snapshot) -> [String: Any]` or something like that

Reply:
Done